README.md

```diff
@@ -1 +1,152 @@
+    on Data Types, to get better performance and flexibility.
+
+
+## Riak
+
+### Install Riak
+
+To install/upgrade **Riak** please follow the instructions in this link:
+[Installing and Upgrading Riak](http://docs.basho.com/riak/latest/ops/building/installing).
+
+### Initial Configurations
+
+Due to the fact that **Riak** comes with default configuration, we need to
+change some parameters required by `sumo_db`.
+
+**Riak** has a main configuration file `riak.conf`, which you can find into
+your installation path `$YOUR_INSTALL_PATH/etc/riak.conf`.
+
+> **Note:** For more information check this link [Configuration Files](http://docs.basho.com/riak/latest/ops/advanced/configs/configuration-files).
+
+First parameter to change is the default **Riak** backend from **Bitcask** to
+**LevelDB**. This change also enables the use of [Riak Secondary Indexes](http://docs.basho.com/riak/latest/ops/advanced/configs/secondary-index/).
+
+    storage_backend = leveldb
+
+Then proceed to enable search capabilities:
+
+    search = on
+
+> **Note:** For more information check this link [Riak Search Settings](http://docs.basho.com/riak/latest/ops/advanced/configs/search/).
+
+### Configuring Riak Data Types and Search
+
+First, let's create and activate a bucket type simply called maps that is set up
+to store Riak maps:
+
+Because `sumo_db_riak` adapter is implemented using **Riak Data Types**, and docs/entities
+are mapped to **Riak Maps**, we need to create a [Riak Bucket Type](http://docs.basho.com/riak/latest/dev/advanced/bucket-types/)
+for those docs that will be stored.
+
+Taking as example our [tests](./test), let's call the bucket type `maps`.
+
+    $ riak-admin bucket-type create maps '{"props":{"datatype":"map"}}'
+    $ riak-admin bucket-type activate maps
+
+Now, let's create a search index called `sumo_test_index` using the default
+schema:
+
+    $ curl -XPUT $RIAK_HOST/search/index/sumo_test_index \
+        -H 'Content-Type: application/json' \
+        -d '{"schema":"_yz_default"}'
+
+With our index created, we can associate our new `sumo_test_index` index with
+our `maps` bucket type:
+
+    $ riak-admin bucket-type update maps '{"props":{"search_index":"sumo_test_index"}}'
+
+Now we can start working with **Riak** from `sumo_db`.
+
+> **Note:** For more information check this link [Riak Data Types and Search](http://docs.basho.com/riak/latest/dev/search/search-data-types/#Maps-Example).
+
+
+## Getting Started
+
+To start use `sumo_db` with this Riak adapter `sumo_db_riak` is pretty easy, you only has to
+follow these steps:
+
+ 1. Add `sumo_db` and `sumo_db_riak` as dependencies in your project.
+
+Using **erlang.mk**:
+
+```erlang
+DEPS = sumo_db sumo_db_riak
+
+dep_sumo_db      = git https://github.com/inaka/sumo_db.git      0.5.0
+dep_sumo_db_riak = git https://github.com/inaka/sumo_db_riak.git 0.0.1
+```
+
+Using **Rebar**:
+
+```erlang
+{deps, [
+  {sumo_db, {git, "https://github.com/inaka/sumo_db.git", {tag, "0.5.0"}}},
```

Comment:
you shouldn't need to do this, this application (sumo_db_riak) should have sumo_db as a dependency already